qemu/tests/x86_cpu_flags.py

```diff
@@ -22,6 +22,9 @@ def run(test, params, env):
     if check_host_flags:
         check_cpu_flags(params, flags, test)
 
+    if params.get("origin_model"):
+        params["cpu_model"] = params["default_cpu_model"]
```

Comment:
Param "default_cpu_model" not set ?

Reply:
> Param "default_cpu_model" not set ?

It is set in avocado-vt cfg so we can just use it.